tests/kani/StdLink/overlapping_traits.rs

```diff
@@ -0,0 +1,49 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+//! This test is to check how Kani handle some std functions. The original issue was:
+//! <https://github.com/model-checking/kani/issues/564>
+//! This code was extracted from Rust by Example book.
+trait UsernameWidget {
+    // Get the selected username out of this widget
+    fn get(&self) -> String;
+}
+
+trait AgeWidget {
+    // Get the selected age out of this widget
+    fn get(&self) -> u8;
+}
+
+// A form with both a UsernameWidget and an AgeWidget
+struct Form {
+    username: String,
+    age: u8,
+}
+
+impl UsernameWidget for Form {
+    fn get(&self) -> String {
+        self.username.clone()
+    }
+}
+
+impl AgeWidget for Form {
+    fn get(&self) -> u8 {
+        self.age
+    }
+}
+
+#[kani::proof]
+#[kani::unwind(10)]
```

Comment:
Is this annotation needed?

Reply:
Let me double check. I think I just copied from the issue description.